spec/order_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'order'
+
+describe Order do
+  let(:menu) {double :menu, dishes: [dish, dish2] }
+  let(:dish) { double :dish, show_details: 'Pizza: £2.00', price: 2.00}
+  let(:dish2) {double :dish2, show_details: 'Burger: £1.50', price: 1.50}
+  let(:dish3) {double :dish3}
+  it { is_expected.to respond_to :order }
+
+  describe '#add_to_order' do
+
+    it { is_expected.to respond_to :add_to_order }
+
+    it 'should add dish to order' do
+      subject.add_to_order(menu, dish, 1)
+      subject.add_to_order(menu, dish2, 1)
+      expect(subject.order).to eql [dish, dish2]
+    end
+
+    it 'should raise error if dish is not on menu' do
+      expect{subject.add_to_order(menu, dish3, 1)}.to raise_error(
+        'Item not on menu')
+    end
+  end
+
+  describe '#show_order' do
+
+    it {is_expected.to respond_to :show_order }
+
+    it 'should show order' do
+      subject.add_to_order(menu, dish, 3)
+      expect(subject.show_order).to eql "\nPizza: £2.00 X3"
+    end
+  end
+
+  describe '#total_price' do
+
+    it { is_expected.to respond_to :total_price }
```

Comment:
These individual expectations are not necessary: you test the method each time later on (here, you test this on line 42). You should avoid duplicating test functionality as this will slow tests down without increasing coverage.